### PR TITLE
Webpack-cli upgraded to v4.0.0

### LINKED ISSUE #141: Upgrade webpack cli to v4
And make the build work again.

https://github.com/meilisearch/docs-searchbar.js/pull/138/checks?check_run_id=1242282848

### PR DESCRIPTION
**Pull request by kakawajazz**
* updated webpack-cli version (4.0.0)
* updated babel packages
* updated .babelrc
* updated build-js shell script

Fixes #141

### CODE REVIEW

**Comment by curquiza** [comment]
Hello @kakawajazz! Thanks a lot for your PR! I requested @bidoubiwa as reviewer 🙂

**Tips**
I see you do this PR from **your** `master` branch which we do not recommend 🙂 For example, if we have refused your PR, you would have had a broken `master` branch compared to ours. Let's keep your PR as it is now, it's okay, but next time, I recommend you creating a separate branch 😉
See here our [contribution guidelines](https://github.com/meilisearch/docs-searchbar.js/blob/master/CONTRIBUTING.md#how-to-contribute).

**Comment by curquiza** [comment]
Hello again @kakawajazz!

I have errors when using your changes.

```bash
$ yarn
$ yarn build
$ yarn serve
[...]
```

`yarn serve` runs a playground that tests if the built package works. But here what I get:

Capture d’écran 2020-10-13 à 10 05 59

Maybe @bidoubiwa can help with what's wrong...?

**Comment by kakawajazz** [comment]
Hello @curquiza!

Thanks for your advice about the branch.

What about your console errors, it looks quite strange. I did just the same.

1. Run commands
```
yarn
yarn build
yarn serve 
```
2. Await till "Compilation finished" log appears
3. Click on link.
image


And here the result in Google Chrome 85.0.4183.121
image

and Firefox 81.0.1
image

**Comment by curquiza** [comment]
I have the same on Firefox

Capture d’écran 2020-10-13 à 12 59 59

but I still get the errors.

Do you?

**Comment by kakawajazz** [comment]
@curquiza Sorry for late answer.

Well, yeah, now I see. Didn't realize first that you are talking about playground.html, not about `playground` in general.

Fixed webpack config file.

**Review by curquiza — APPROVED** [review]
It works now thanks a lot!!
So LGTM 👍
But still waiting for the @bidoubiwa review before merging 🙂

File: webpack.config.jsdelivr.babel.js
Comment (bidoubiwa) [inline webpack.config.jsdelivr.babel.js:20]: Hey sorry if my question is a bit unaware but why is it `modules: cjs`? Is this not a browser library?

**Review by bidoubiwa — COMMENTED** [review]
Hey @kakawajazz thanks a lot for your PR!

File: webpack.config.jsdelivr.babel.js
Comment (curquiza) [reply webpack.config.jsdelivr.babel.js:20]: It's indeed only used in the browser.

Comment (curquiza) [reply webpack.config.jsdelivr.babel.js:20]: Should we change the `cjs` @kakawajazz?

**Comment by curquiza** [comment]
Close this PR due to no activities.
The webpack cli is now updated thanks to #159.
This PR still might be an inspiration to upgrade babel-loader.

**Pull request closed without merging**
